doc/source/.res/code-block/bash/pbm-delete-backup-older-than-backup.txt

```diff
@@ -0,0 +1,12 @@
+.. code-block:: bash
+
+ #Get the backup name
+ $ pbm list
+ Backup history:
+   2020-04-20T13:13:20Z
+   2020-04-20T13:07:34Z
+   2020-04-20T10:55:42Z
+ #Delete backups older than the specified one
+ $ pbm delete-backup -f --older-than 2020-04-20T13:13:20Z
```

Comment:
`pbm --older-than` doesn't delete backups in relation to another backup.`--older-than` works only with timestamps in one of two formats: `2020-04-20T13:13:20`(! without `Z` in the end) or `2020-04-20`.